File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        # if len(s) != len(t):
        #     return False

        char = {}

        for i in s:
            char[i] = char.get(i, 0) + 1
        
        for j in t:
            if j not in char or char[j] == 0:
                return False
            char[j] -= 1
        
        for v in char.values():
            if v != 0:
                return False
        
        return True

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    removeDuplicates(nums) {
        let startIndex = 1

        for(let endIndex=1; endIndex < nums.length; endIndex++) {
            if (nums[endIndex] !== nums[endIndex - 1]) {
                nums[startIndex] = nums[endIndex]
                startIndex++
            }
        }

        return startIndex
    }
}
